src/modules/getData.js

```diff
@@ -0,0 +1,7 @@
+// FECH ALL SCORES
+const fetchData = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Zl4d7IVkemOTTVg2fUdz/scores/');
```

Comment:
Kindly make sure you are creating a new game ID for your project. You are using the game ID provided as an example in the documentation that's why your project has a lot of unnecessary scores. Kindly create a new game ID this is a project requirement. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/202595906-d9771998-5bd0-4216-99e1-9995f00ef681.png)
![image](https://user-images.githubusercontent.com/98363075/202596151-0011efaa-a3ee-423f-98f2-d818b7936bb2.png)